packages/protocol/test/mocks/MockAddressManager.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+/// @title Mock Address Manager
+/// @notice Provides a simplified mock-up of an address manager for testing purposes,
+/// substituting for a more complex AddressManager in test environments.
+/// @dev This mock manager is used in tests to simulate interactions with a vault system
+/// that typically involves managing addresses of various contracts like an ERC20Vault.
+contract MockAddressManager {
```

Comment:
actually you don't need a mock address manager, you can deploy a real one then use the admin to register a address for a name